### DISCUSSION TITLE
Account suspended, support unresponsive

### DISCUSSION THREAD

**Discussion by laszloh2** [community/community#118130] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi,
My account was suspended without any reason given on 20.02.2024. I just got a mail, that suspicious activity was deteced and I should change my password. When I tried to do just that 30 minutes later, the account was already  suspended. I am not aware of any missconduct from my side.

I raised a ticket on the 20th of February, which is with today almost 3 months without any replies (and yes, I looked into my spam folder :) ). I would understand 2 weeks of delay, but almost 50 days seems really excessive for a response.

Account: laszloh
Ticket: #2610609

**Comment by jfullstackdev** [comment]
is this same account you are using to post ? or this is a new one ? can you give me the old suspended account, so I can check it too ? though I'm not a GitHub Staff, just want to help

**Reply by mateusloubach** [reply]
Banned or Spam tagged accounts typically aren't able to be found available until they remove the ban or the tag.

This account I'm using is an example. Til yesterday at least, you couldn't even be redirected to this account. not sure if it's still happening, but it's still tagged as spam, so, give it a try.

**Reply by laszloh2** [reply]
No, this is a fallback I created 2 weeks after the suspension. The suspended account is [@laszloh](https://github.com/laszloh/)

**Reply by jfullstackdev** [reply]
so I checked that account it says 404 but why would you want to restore it and not simply start anew ? do you have significant reason ? again I'm not a GitHub Staff, just helping, you can always start with your new account

**Reply by laszloh2** [reply]
And lose around 8 years of development history and repositories of which I don't have backups? I think not, thank you for your lip service.

**Comment by mateusloubach** [comment]
As per the ban or spam tag, it's just how Github faces these issues. I'm sure you can check the general tab, and you'll find many others are facing the same issues. Unfortunately, Github doesn't give priority to this. You could probably tag them in a post on Linkedin or something of the sorts, but it's not anything guaranteed.

**Comment by PetarTuovic** [comment]
I can confirm as well that Support is currently terrible... I am waiting for an answer on my locked account which has a Copilot subscription, on how to transfer that subscription to at least my new account (currently in use) or to cancel it... I am paying for a feature that I can't use at this moment. This is really unacceptable!

**Comment by martoc** [comment]
My account has been flagged for more than 12 days without response from support, I've been charged for copilot, github actions and pro tier, it has been a bit disappointing that the support ticket hasn't been even seen yet, I don't know what to do my account is 11 years old and I've never had this problem, please help

**Comment by ghost** [comment]
@laszloh2 
Did you get a response? Mine got suspended too, it has been more than a month and still no response.
ticket: #2719793

**Reply by ghost** [reply]
Did you get your account back? Did you create another?

**Reply by laszloh2** [reply]
I got an answer from support and got the account back after ~110 days of wait time.

**Comment by asmawi117** [comment]
I raised a ticket on February 13, but I still haven't received a reply.

**Reply by asmawi117** [reply]
The support team finally replied to me after 114 days, suggesting that I contact them or forward the message using the email associated with my hacked account. I have not heard back from them since, and they also closed my ticket. The issue still hasn't been resolved.

**Comment by laszloh2** [comment]
Nope, 97 days without a response. I'm really wondering if there even are any people in support, or if it's just a black hole.

**Comment by clearskyaaa** [comment]
3 weeks without a response,put important code on github was a bad decision :(

**Comment by ghost** [comment]
Same here

**Comment by rlzyy** [comment]
Info HT

**Comment by ghost** [comment]
My account got flagged without any e-mails. This means no one can see my profile publicly. It's been a month since. Waiting for their response.

**Comment by alanzimm85** [comment]
Please help, i need to get my account back, i sent a lot of curriculums with projects on this github account. I didn´t do nothing wrong. It's not from this account, i can´t acces the other. Ticket ID: 2921645

You have been suspended from GitHub.
 
This moderation action was taken after we received a report and determined that your account violates our Acceptable Use Policy prohibiting the use of our platform in direct support of unlawful attacks that cause technical harms. We do not allow anyone to use our platform in direct support of unlawful attacks that cause technical harms, such as using GitHub as a means to deliver malicious executables or as attack infrastructure, for example by organizing denial of service attacks or managing command and control servers. Technical harms means overconsumption of resources, physical damage, downtime, denial of service, or data loss, with no implicit or explicit dual-use purpose prior to the abuse occurring.

When making content moderation decisions, we consider information from a variety of sources, including: account profile data, information contained in submitted reports/notices or discovered through our own voluntarily initiated investigations, and context around the contents of the repository.

If you wish to regain access or would like to dispute that a violation occurred and can provide additional information to show that a different decision should have been reached, please review our [Appeal and Reinstatement Policy](https://docs.github.com/en/site-policy/acceptable-use-policies/github-appeal-and-reinstatement) and submit a request via our [form](https://support.github.com/contact/reinstatement).
